dashboard/src/actions/apps.test.tsx

```diff
@@ -65,3 +65,34 @@ describe("fetches applications", () => {
     });
   });
 });
+
+describe("delete applications", () => {
+  const deleteApp = App.delete;
+  let deleteAppMock: jest.Mock;
+  beforeEach(() => {
+    App.delete = jest.fn(() => []);
```

Comment:
as suggested in https://github.com/kubeapps/kubeapps/pull/563#discussion_r214468729, I think would be more readable as:

```
deleteAppMock = jest.fn(() => []);
App.delete deleteAppMock;
```